dali/dafilesrv/dafilesrv.cpp

```diff
@@ -34,7 +34,7 @@
 
 static const bool defaultRowServiceOnStdPort = true;
 static const bool defaultDedicatedRowServiceSSL = false;
-
+static const char* defaultRowSericeConfiguration = "RowSvc";
```

Comment:
this change seems somewhat unrelated

Reply:
yes. will remove

Reply:
doesn't look like you removed it

Reply:
The variable is used by the code in the the source file, so we can't remove it.